sw/device/silicon_creator/rom/data/rom_manual_testplan.hjson

```diff
@@ -25,5 +25,27 @@
       stage: V2
       tests: []
     }
+    {
+      name: rom_manual_cpuctrl_layout
+      desc: '''Verify that the mask for the `cputctrl` CSR in `rom_init()` is up to date.
+
+            See [Ibex documentation][1] for details.
+
+            - Verify that the first six bits of the `CPUCTRL` CSR match what
+              `rom_init()` and `CREATOR_SW_CFG_CPUCTRL` OTP item expect:
```

Comment:
nit: TODO cross-reference OTP documentation describing the bit-8 masking behavior.

Reply:
We actually write only bits 5:0 to `cpuctrl` so I added a comment for this to `otp_ctrl_img_creator_sw_cfg.hjson`. The hardened check that masks bit 8 but includes bit 7 and 6 is an implementation detail that doesn't have to go into the documentation, I think.